index.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable import/extensions */
+import ui from './modules/ui.js';
+import store from './modules/store.js';
+// eslint-disable-next-line
+import { DateTime } from './modules/luxon.js';
+
+class Books {
+  constructor(Title, Author) {
+    this.title = Title;
+    this.author = Author;
+  }
+}
+
+// add book//
+document.addEventListener('DOMContentLoaded', ui.display());
+document.querySelector('#form').addEventListener('submit', () => {
+  const Title = document.querySelector('#title').value;
+  const Author = document.querySelector('#author').value;
+  const book = new Books(Title, Author);
+  store.setBook(book);
+  ui.clearFields();
+});
+// delete Book//
+const remove = document.querySelector('.bookList');
+remove.addEventListener('click', (e) => {
+  store.deleteBooks(e.target.previousElementSibling.firstElementChild.textContent);
+  // eslint-disable-next-line
+  location.reload();
+});
+
+const List = document.querySelector('.List');
+const Add = document.querySelector('.Add');
+const Contact = document.querySelector('.Contact');
+
+Add.addEventListener('click', () => {
+  document.querySelector('.book-list-container').style.display = 'none';
+  document.querySelector('.input').style.display = 'block';
+  document.querySelector('.contact').style.display = 'none';
+});
+
+List.addEventListener('click', () => {
+  document.querySelector('.book-list-container').style.display = 'block';
+  document.querySelector('.input').style.display = 'none';
+  document.querySelector('.contact').style.display = 'none';
+  // eslint-disable-next-line
+  location.reload();
+});
+
+Contact.addEventListener('click', () => {
+  document.querySelector('.book-list-container').style.display = 'none';
+  document.querySelector('.input').style.display = 'none';
+  document.querySelector('.contact').style.display = 'block';
+});
```

Comment:
- [ ] [_OPTIONAL_] Dividing these pieces of code into small reusable functions would be a nice idea since that will make your code a whole lot cleaner and more maintainable (DRY principles). Additionally it would help you out a lot to have small pure functions when you get to testing your application which you will do in the near future. Kindly divide this code into small reusable functions if you can (ES6 syntax recommended)

Reply:
this is a fast solution I found to be capable of removing books from the UI, without having to use a function,
but still, for some reason, Eslint doesn't allow the use of (location. reload)
don't u think it's worth being used there ?